src/cli/cli.cpp

```diff
@@ -361,29 +359,29 @@ int Interpreter::Hex2Bin(const char *aHex, uint8_t *aBin, uint16_t aBinLength, b
     return rval;
 }
 
-void Interpreter::AppendResult(otError aError) const
+void Interpreter::AppendResult(otError aError)
```

Comment:
Can it be still `const`?

Reply:
no, the Output() function changes the internal state.